spec/factories/payments.rb

```diff
@@ -1,13 +1,17 @@
 FactoryBot.define do
   factory :payment do
     transient do
-      claim_policy { StudentLoans }
+      claim_policies { [StudentLoans] }
     end
 
-    claim { association(:claim, :approved, policy: claim_policy) }
+    claims do
+      claim_policies.map do |policy|
+        association(:claim, :approved, policy: policy)
+      end
+    end
     association(:payroll_run, factory: :payroll_run)
 
-    award_amount { claim.award_amount }
+    award_amount { claims.sum(&:award_amount) }
```

Comment:
Looking at the payment mailer previews I found it odd that the amount "you will receive" was totally out of whack from the award amount, which looks really confusing. Do you think we should update the factory so the hard-coded values in the `:with_figures` trait are generated based on the `award_amount` instead to make these factory-generated payments more realistic?

Reply:
Good shout. Adding some rough calculations to approximate Cantium's grossing up.

Reply:
👍